test/unit/mdc-slider/mdc-slider.test.js

```diff
@@ -29,11 +29,15 @@ suite('MDCSlider');
 
 function getFixture() {
   return bel`
-    <div class="mdc-slider" tabindex="0" role="slider" aria-valuemin="0" aria-valuemax="100" aria-valuenow="0">
+    <div class="mdc-slider foo" tabindex="0" role="slider" aria-valuemin="0" aria-valuemax="100" aria-valuenow="0">
```

Comment:
Add any extraneous class names in the individual test using `root.classList.add('foo')` when you're setting up the test. The fixture should be as generic as possible, and any test that needs `foo` present should exist in isolation.